samples/xlsx-import+svelte/README.md

```diff
@@ -0,0 +1,47 @@
+# Svelte
+
+This is an example how to use `xlsx-import` in Svelte.
+
+**LIB Version:** `2.3.4`
+
+## Scripts
+
+`npm run dev` - compiles and hot-reloads for development
```

Comment:
For the purposes of being consistent with other examples, please change the `start` script in package.json into `"start": "npm run dev"`, and lists it here instead of the `dev` script.